.azure/pipelines/jobs/default-build.yml

```diff
@@ -146,6 +146,8 @@ jobs:
       - LC_ALL: 'en_US.UTF-8'
       - LANG: 'en_US.UTF-8'
       - LANGUAGE: 'en_US.UTF-8'
+      - ${{ if ne(parameters.disableComponentGovernance, '') }}:
+        disableComponentGovernance: ${{ parameters.disableComponentGovernance }}
```

Comment:
This won't work here. Needs to be up above i.e. immediately above the `variables:` line. That'll change the indentation too.

Reply:
Put another way, this is about parameters (to the job.yml template) and not variables